distributed/tests/test_worker.py

```diff
@@ -1586,3 +1586,15 @@ def bad_startup(w):
             w = await Worker(s.address, startup_information={"bad": bad_startup})
         except Exception:
             pytest.fail("Startup exception was raised")
+
+
+@pytest.mark.asyncio
+async def test_update_latency():
```

Comment:
```suggestion
async def test_update_latency(cleanup):
```

This will enforce a variety of checks.

In particular, it will also require that your schedulers and workers are cleaned up by the end of the test.  I recommend using `async with` context managers below for both.

Reply:
done